demo.rhm

```diff
@@ -216,7 +216,7 @@ bind.infoer 'build_reverse_cons_infoer($in_id, ($a_in, $b_in))':
   | '($a_ann, $a_id, $a_info, ($a_bind_info ...), $a_matcher, $a_binder, $a_data)':
       match bind_ct.unpack_info(b)
       | '($b_ann, $b_id, $b_info, ($b_bind_info ...), $b_matcher, $b_binder, $b_data)':
-          '($("matching((_ :: " & unwrap_syntax(a_ann) & ") <> (_ :: " & unwrap_syntax(b_ann) & "))"),
+          '($("matching((_ :: " +& unwrap_syntax(a_ann) +& ") <> (_ :: " +& unwrap_syntax(b_ann) +& "))"),
```

Comment:
If we all agree that `&` should be changed to `+&` to make room for the rest operator, can we make this change as a separate PR? I think it unnecessarily makes this proposal difficult to read.

Reply:
Sure. I made it as a separate commit so I can do that.

Reply:
Okay, I made a separate PR: https://github.com/racket/rhombus-prototype/pull/237